spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/config/EurekaDiscoveryClientConfigServiceBootstrapConfiguration.java

```diff
@@ -36,13 +52,52 @@
 @ConditionalOnProperty(value = "spring.cloud.config.discovery.enabled",
 		matchIfMissing = false)
 @Configuration(proxyBeanMethods = false)
-@Import({ EurekaDiscoveryClientConfiguration.class, // this emulates
-		// @EnableDiscoveryClient, the import
-		// selector doesn't run before the
-		// bootstrap phase
-		EurekaClientAutoConfiguration.class,
-		EurekaReactiveDiscoveryClientConfiguration.class,
-		ReactiveCommonsClientAutoConfiguration.class })
 public class EurekaDiscoveryClientConfigServiceBootstrapConfiguration {
 
+	@Bean
+	@ConditionalOnMissingBean(value = EurekaClientConfig.class,
+			search = SearchStrategy.CURRENT)
+	public EurekaClientConfigBean eurekaClientConfigBean() {
+		return new EurekaClientConfigBean();
+	}
+
+	@Bean
+	@ConditionalOnMissingBean
+	public WebClientEurekaHttpClient configDiscoveryEurekaHttpClient(
+			EurekaClientConfigBean config) {
+		List<String> urls = EndpointUtils.getServiceUrlsFromConfig(config, "unknown",
```

Comment:
@twicksell or @troshko111 I put "unkown" here, is there a better default?

Reply:
Based on https://github.com/Netflix/eureka/blob/master/eureka-client/src/main/java/com/netflix/discovery/endpoint/EndpointUtils.java#L373 seems like there's no functional difference but specifying `default` here will avoid that warn log.

Reply:
Thanks!